internal/pkg/service/stream/storage/node/coordinator/fileimport/operator.go

```diff
@@ -362,6 +362,16 @@ func (o *operator) doImportFile(ctx context.Context, lock *etcdop.Mutex, file *f
 	// Empty file import can be skipped in the import implementation.
 	err = o.plugins.ImportFile(ctx, file.File, stats.Staging)
 	if err != nil {
+		// Record metric for failed file imports
+		if file.Retry.RetryAttempt < 4 {
+			counter := o.metrics.FileImportFailed
+			attrs := append(
+				file.FileKey.SinkKey.Telemetry(),
+				attribute.String("operation", "fileimport"),
+			)
+			counter.Add(ctx, 1, metric.WithAttributes(attrs...))
```

Comment:
The variable looks unnecessary to me.

```suggestion
			attrs := append(
				file.FileKey.SinkKey.Telemetry(),
				attribute.String("operation", "fileimport"),
			)
			o.metrics.FileImportFailed.Add(ctx, 1, metric.WithAttributes(attrs...))
```

Reply:
[4c45268](https://github.com/keboola/keboola-as-code/pull/2287/commits/4c45268af7ae0c949612a0e343001ea68b2686de)